ttnn/ttnn/operations/unary_backward.py

```diff
@@ -165,36 +146,36 @@ def _golden_function_backward_with_reverse_string(
 
 ttnn.attach_golden_function(
     ttnn.hardshrink_bw,
-    golden_function=lambda grad, input, alpha=None, *args, **kwargs: _golden_function_unary_backward_with_float(
-        torch.hardshrink, grad, input, alpha, *args, **kwargs
+    golden_function=lambda grad, input, alpha=0.5, *args, **kwargs: _golden_function_unary_backward_with_float(
+        torch.hardshrink, grad, input, lambd=alpha, *args, **kwargs
     ),
 )
 
 ttnn.attach_golden_function(
     ttnn.softshrink_bw,
-    golden_function=lambda grad, input, alpha=None, *args, **kwargs: _golden_function_unary_backward_with_float(
-        torch.softshrink, grad, input, alpha, *args, **kwargs
+    golden_function=lambda grad, input, alpha=0.5, *args, **kwargs: _golden_function_unary_backward_with_float(
+        torch.nn.functional.softshrink, grad, input, lambd=alpha, *args, **kwargs
```

Comment:
lambd?

Reply:
Thats the keyword argument for `softshrink`
<img width="731" alt="Screenshot 2024-10-22 at 10 20 17 AM" src="https://github.com/user-attachments/assets/91867595-ae03-4e49-9eab-8256d2ed06c0">